<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartingPanel</class>
 <widget class="QWidget" name="StartingPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audiobook Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/SVG/svg/icon/AudiobookEditorLogo@16.svg</normaloff>:/SVG/svg/icon/AudiobookEditorLogo@16.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* starting panel */


position: relative;
width: 640px;
height: 480px;

background: #333333;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Rectangle 39 */


position: absolute;
width: 60px;
height: 60px;
left: 290px;
top: 100px;

background: rgba(10, 110, 202, 0.3);
border-radius: 30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/AudiobookEditorLogo@16.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>186</y>
     <width>174</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position: static;
width: 174px;
height: 28px;
left: 73px;
top: 16px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: bold;
font-size: 20px;
line-height: 27px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;

color: #FFFFFF;


/* Inside Auto Layout */

flex: none;
order: 0;
flex-grow: 0;
margin: 0px 0px;</string>
   </property>
   <property name="text">
    <string>Audiobook Editor</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>238</y>
     <width>544</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* This is a public prototype conforming and demostrating the Audiobooks. W3C Recommendation 10 November 2020. Have fun :) */


position: static;
width: 504px;
height: 30px;
left: 48px;
top: 8px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 15px;
text-align: center;

color: rgba(255, 255, 255, 0.6);


/* Inside Auto Layout */

flex: none;
order: 0;
flex-grow: 1;
margin:0px 0px;</string>
   </property>
   <property name="text">
    <string>This is a public prototype conforming and demostrating the Audiobooks W3C Recommendation 10 November 2020. Have fun :)</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>338</y>
     <width>194</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: static;
width: 194px;
height: 24px;
left: 60px;
top: 6px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 18px;
display: flex;
align-items: center;

/* icon/primary */

color: #5AB0FF;

/* Inside Auto Layout */

flex: none;
order: 1;
align-self: stretch;
flex-grow: 0;
margin: 6px 0px;</string>
   </property>
   <property name="text">
    <string>Create a new audiobook project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>374</y>
     <width>194</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: static;
width: 194px;
height: 24px;
left: 60px;
top: 6px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 18px;
display: flex;
align-items: center;

/* icon/primary */

color: #5AB0FF;

/* Inside Auto Layout */

flex: none;
order: 1;
align-self: stretch;
flex-grow: 0;
margin: 6px 0px;</string>
   </property>
   <property name="text">
    <string>Open a file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_New">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>344</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/welcome-new.svg</normaloff>:/SVG/svg/icon/welcome-new.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Open_File">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>380</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/welcome-file.svg</normaloff>:/SVG/svg/icon/welcome-file.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>410</y>
     <width>194</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: static;
width: 194px;
height: 24px;
left: 60px;
top: 6px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 18px;
display: flex;
align-items: center;

/* icon/primary */

color: #5AB0FF;

/* Inside Auto Layout */

flex: none;
order: 1;
align-self: stretch;
flex-grow: 0;
margin: 6px 0px;</string>
   </property>
   <property name="text">
    <string>Open a project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Open_Project">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>416</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/welcome-folder.svg</normaloff>:/SVG/svg/icon/welcome-folder.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Close">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>6</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/starting-panel-close.svg</normaloff>:/SVG/svg/icon/starting-panel-close.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
